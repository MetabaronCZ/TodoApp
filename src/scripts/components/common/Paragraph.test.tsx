import React from 'react';

import { render } from '@testing-library/react';
import { describe, expect, it } from '@jest/globals';

import { Paragraph } from 'components/common/Paragraph';
import { withMockedProviders } from 'test/component';

describe('components/common/Paragraph', () => {
  it('should render correctly', () => {
    const tree = render(withMockedProviders(<Paragraph>CONTENT</Paragraph>));
    expect(tree.container).toMatchSnapshot();
    expect(tree.getByText('CONTENT')).toBeInTheDocument();
  });
});
