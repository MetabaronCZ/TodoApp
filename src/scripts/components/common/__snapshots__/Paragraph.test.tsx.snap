// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/common/Paragraph > should render correctly 1`] = `
.c0 {
  font-family: Open Sans,Arial,sans-serif;
  color: #222;
  font-size: 14px;
  line-height: 20px;
}

<div>
  <p
    class="c0"
  >
    CONTENT
  </p>
</div>
`;
